config/environments/production.rb

```diff
@@ -46,9 +48,9 @@
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   # config.force_ssl = true
 
-  # Use the lowest log level to ensure availability of diagnostic information
-  # when problems arise.
-  config.log_level = :debug
+  # Include generic and useful information about system operation, but avoid logging too much
+  # information to avoid inadvertent exposure of personally identifiable information (PII).
+  config.log_level = :info
```

Comment:
wondering if it won't log too much information. `:warn` could be better.

Reply:
That is also a default, but `warn` is an option.